### PR TITLE
cve_update

### PR DESCRIPTION
**Pull request by pj-spoelders**

### CODE REVIEW

**Comment by pj-spoelders** [comment]
Closes #13

File: .github/workflows/release_droid_upload_github_release_assets.yml [hunk 1]
```diff
@@ -24,7 +24,9 @@ jobs:
       - name: Build with Maven skipping tests
         run: mvn --batch-mode clean verify -DskipTests
       - name: Generate sha256sum files
-        run: find target -maxdepth 1 -name *.jar -exec bash -c 'sha256sum {} > {}.sha256' \;
+        run: |
+          cd target
+          find . -maxdepth 1 -name \*.jar -exec bash -c 'sha256sum {} > {}.sha256' \;
```
Comment (ckunki) [inline .github/workflows/release_droid_upload_github_release_assets.yml:29]: Proposal: mark as generated file in `.gitattributes`

**Review by ckunki — APPROVED** [review]

File: .github/workflows/release_droid_upload_github_release_assets.yml (hunk 1 above)
Comment (pj-spoelders) [reply .github/workflows/release_droid_upload_github_release_assets.yml:29]: done

File: pom.xml [hunk 2]
```diff
@@ -209,6 +209,9 @@
                         <exclude>CVE-2022-24823</exclude>
                         <exclude>CVE-2022-2047</exclude>
                         <exclude>CVE-2022-30187</exclude>
+                        <!-- org.yaml.snakeyaml (1.33) https://ossindex.sonatype.org/vulnerability/CVE-2022-38750
+                        Currently not fixed in latest version, however a test dependency and not used on untrusted files.
+                        Can be removed once org.yaml.snakeyaml upgrades to a newer version. -->
```
Comment (ckunki) [inline pom.xml:214]: Only as (optional) comment: False positive in snakeyaml. According to https://bitbucket.org/snakeyaml/snakeyaml/issues/531/stackoverflow-oss-fuzz-47081 this is already fixed in 1.32.

**Review by ckunki — APPROVED** [review]

**Pull request merged**